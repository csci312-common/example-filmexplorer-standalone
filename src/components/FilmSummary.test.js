import React from 'react';
import { shallow } from 'enzyme';

import FilmSummary from './FilmSummary';

const film = {
  id: 135397,
  overview: 'Twenty-two years after the events of Jurassic Park...',
  release_date: '2015-06-12',
  poster_path: '/jjBgi2r5cRt36xF6iNUEhzscEcb.jpg',
  title: 'Jurassic World',
  vote_average: 6.9
};

describe('FilmSummary', () => {
  test('FilmSummary snapshot', () => {
    const comp = shallow(
      <FilmSummary {...film} onClick={jest.fn} setRatingFor={jest.fn} />
    );
    expect(comp).toMatchSnapshot();
  });
});
