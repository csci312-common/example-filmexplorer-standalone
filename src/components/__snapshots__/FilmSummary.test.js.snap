// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmSummary FilmSummary snapshot 1`] = `
<styled.div>
  <FilmTitle
    onClick={[Function]}
  >
    Jurassic World
  </FilmTitle>
  <StarRating
    rating={0}
    setRating={[Function]}
  />
  <styled.p>
    (
    2015
    )
  </styled.p>
  <styled.p>
    TMDB score: 
    6.9
  </styled.p>
</styled.div>
`;
